flake.nix

```diff
@@ -40,7 +41,13 @@
             inherit config extraModules extraSpecialArgs home-manager-path pkgs;
             isFlake = true;
           };
-      apps.nix-on-droid.aarch64-linux = app;
-      defaultApp.aarch64-linux = app;
-    };
+
+      apps.aarch64-linux = {
+        default = app;
+        nix-on-droid = app;
+      };
+    }
+    // flake-utils.lib.eachDefaultSystem (system: {
+      formatter = nixpkgs.legacyPackages.${system}.nixpkgs-fmt;
```

Comment:
Nit: why do we export it?

Reply:
nix flake show won't show it, but it is necessary for nix fmt to work.

Reply:
Huh, OK then.